addons/sale_timesheet/models/project_update.py

```diff
@@ -71,15 +72,35 @@ def _get_profitability_values(self, project):
         if not (self.user_has_groups('project.group_project_manager') and costs_revenues):
             return {}
         profitability_items = project._get_profitability_items(False)
+        for revenue in profitability_items['revenues']['data']:
+            revenue['formatted'] = {
```

Comment:
Actually, after discussing it with @xavierbol, we think that it is better to format the values in js, to keep this dict as light as possible - both for readability and not to overload the network.

Reply:
Sorry for not pointing it earlier :(